estate/__init__.py

```diff
@@ -0,0 +1 @@
+from . import models
```

Comment:
```suggestion
from . import models

```
always add new line at the end of every file

Reply:
done